include/variables.h

```diff
@@ -20,6 +20,8 @@ extern ScriptInstance* gWorldScriptList[128];
 extern ScriptInstance* gBattleScriptList[128];
 extern ScriptInstance** gCurrentScriptListPtr[128];
 
+extern s32 gListCount;
```

Comment:
Maybe needs a more descriptive name?

Reply:
Yeah I agree. This is from SR. I changed it to gScriptListCount.